lib/generators/module_component/templates/app/services/configuration.rb.erb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module <%= class_name %>
+  class Configuration < Common::Client::Configuration::REST
+    def base_path
+      Settings.<%= class_name.downcase %>.url
+    end
+
+    def service_name
+      '<%= class_name.upcase %>/HelloWorld'
```

Comment:
Should this be `capitalize` rather than `upcase`? Perhaps `classify`?

Reply:
Ah, yeah I may like classify. This template was just a copy/paste from the previous generator